src/main/scala/tech/cryptonomic/conseil/tezos/TezosNodeOperator.scala

```diff
@@ -377,6 +377,15 @@ class TezosNodeOperator(
     (fetchProposals, fetchBakers, fetchBallots).tupled.run(blocks.filterNot(b => isGenesis(b.data)))
   }
 
+  /** Fetches active delegates from node */
+  def fetchActiveDelegates(blockHashes: List[(Int, BlockHash)]): Future[List[(BlockHash, List[String])]] = {
+    import cats.instances.future._
+    import cats.instances.list._
+    import tech.cryptonomic.conseil.generic.chain.DataFetcher.fetch
+
+    fetch[BlockHash, List[String], Future, List, Throwable].run(blockHashes.filterNot(_._1 == 0).map(_._2))
```

Comment:
This line is kinda cryptic, I'd beg for more expressive naming and trade it over conciseness.

Reply:
ok, changed it to be more expressive